libs/qCC_db/src/ccPolyline.cpp

```diff
@@ -111,7 +111,6 @@ void ccPolyline::importParametersFrom(const ccPolyline& poly)
 	showColors(poly.colorsShown());
 	showVertices(poly.verticesShown());
 	setVertexMarkerWidth(poly.getVertexMarkerWidth());
-	setVisible(poly.isVisible());
```

Comment:
Not sure to understand? Apparently it was missing (if I remember well).

Reply:
`setVisible(poly.isVisible());` was in the old version of importParametersFrom, but it isn't in the new version. 

I thought it might have been removed for testing/development

Reply:
I have it on my side though...  how strange

Reply:
I can see it properly by looking at the file, it was just the diff that looked made it look missing